docs/release-notes.md

```diff
@@ -233,24 +126,19 @@ v1.0.0 - 2022-01-07
 - **[Poor error msg related WiFi profile issues](https://github.com/open-amt-cloud-toolkit/rps/issues/594):** Enhancement
 - **[RPS not able to remove wifi profile from already configured deviceEnhancement](https://github.com/open-amt-cloud-toolkit/rps/issues/597):** Bug 
 #### MPS
-- **CIM_PhysicalPackage may not return all results compared to v2.1**
-- **[Direct Connection from MPS to AMT](https://github.com/open-amt-cloud-toolkit/mps/issues/10):** Enhancement
 - **[Should return error on additional KVM connections for a single device](https://github.com/open-amt-cloud-toolkit/mps/issues/104):** Enhancement
 - **[AMT does not connect to MPS after configuration](https://github.com/open-amt-cloud-toolkit/mps/issues/300):** Known Issue
 - **[Use database abstraction/ORM layer to support multiple SQL-based database](https://github.com/open-amt-cloud-toolkit/mps/issues/360):** Enhancement
 - **[Feature Request: Create configuration parameter to disable "Auth" Service from MPS](https://github.com/open-amt-cloud-toolkit/mps/issues/439):** Enhancement
 - **[CIRA connection getting dropped randomly](https://github.com/open-amt-cloud-toolkit/mps/issues/441):** Bug
 - **[Short Lived Bearer Token for KVM Session Support](https://github.com/open-amt-cloud-toolkit/mps/issues/527):** Enhancement
-- **[Allow filtering devices by hostname](https://github.com/open-amt-cloud-toolkit/mps/issues/548):** Enhancement
 #### RPC
 - **[rpc-go missing some output compared to the c version, and intermittently misses other pieces](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/25):** Bug
-- **[both c rpc and rpc-go seem to be missing the DNS Suffix set in MEBx](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/26):** Needs More Investigation
 - **[activation failures would benefit from passing the RPS error code to the client](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/27):** Enhancement
 - **[Gosh it would be excellent if rpc could tell the user that they don't have an AMT compatible network device](https://github.com/open-amt-cloud-toolkit/rpc-go/issues/27):** Enhancement
```

Comment:
'Gosh it would be excellent....', Looks like it links to the incorrect Issue.  Points to the same one as above.